src/openrct2/util/Util.h

```diff
@@ -20,7 +20,9 @@
 
 int32_t squaredmetres_to_squaredfeet(int32_t squaredMetres);
 int32_t metres_to_feet(int32_t metres);
+int32_t feet_to_metres(int32_t feet);
 int32_t mph_to_kmph(int32_t mph);
+int32_t kmph_to_mph(int32_t kmph);
```

Comment:
These should never be needed. Always work in terms of mph and meters.

Reply:
Yes keep it simple mph / meters.

Reply:
If you want me to do that, I'll do that, but I think it'd be really inconsistent to type in a speed in MPH when your settings say you're using metric. It's not like with the ride launch speed, with a spinner, where you can just transform the MPH to show kmph, it's literally typing in the value, so no trickery possible to alter it.